src/index.js

```diff
@@ -44,8 +44,8 @@ async function run() {
   core.info('Fetching changed files in the pull request');
   const changed_files = await github.fetch_changed_files();
 
-  core.info('Fetching currently requested reviewers');
-  const current_reviewers = await github.fetch_current_reviewers();
+  core.info('Fetching previously requested reviewers');
+  const current_reviewers = await github.fetch_reviewers();
```

Comment:
```suggestion
  core.info('Fetching requested reviewers');
  const current_reviewers = await github.fetch_reviewers();
```

I wonder if it would make more sense to omit "previously" here. I feel like, when fetching the full timeline, we are really fetching "current _and_ previous" reviewers?

Reply:
I put `requested_reviewers` since `reviewers` is already taken, that should remove the connotation that it was `previous` reviewers and not both `previous` and `current`